packages/wonder-blocks-timing/src/util/interval.ts

```diff
@@ -15,7 +15,7 @@ import type {IInterval, SchedulePolicy, ClearPolicy} from "./types";
  * @implements {IInterval}
  */
 export default class Interval implements IInterval {
-    _intervalId: number | null | undefined;
+    _intervalId: ReturnType<typeof setInterval> | null | undefined;
```

Comment:
**note**: This was a tricky one. The global `setInterval` has a return type of `NodeJS.Timer`. This is what is likely correct for in tests where we're not in a JSDOM, but the `window.setInterval` call that will run in browsers is typed to return a number. 

The easy fix to satisfy TS without a functional change is what we have here.

Reply:
neat